### PR TITLE
Additional parameters to control segmentation inference.

### PR DESCRIPTION
**Pull request by FedorKononov**
* Allow masks to spill over detection boxes boundaries
* Always return mask logits

image

### CODE REVIEW

**Review by christopherkuemmel — APPROVED** [review]
Looks fine to me. Though I haven't tested it myself. Maybe @milanilic322 can have a more experienced look on this?
